akd_local_auditor/src/console_log.rs

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+//
+// This source code is licensed under both the MIT license found in the
+// LICENSE-MIT file in the root directory of this source tree and the Apache
+// License, Version 2.0 found in the LICENSE-APACHE file in the root directory
+// of this source tree.
+
+use colored::*;
+use log::Level;
+use log::Metadata;
+use log::Record;
+use once_cell::sync::OnceCell;
+use std::io::Write;
+use tokio::time::Duration;
+use tokio::time::Instant;
+
+static EPOCH: OnceCell<Instant> = OnceCell::new();
+
+pub struct ConsoleLogger {
+    pub level: Level,
+}
+
+impl ConsoleLogger {
+    pub fn touch() {
+        EPOCH.get_or_init(Instant::now);
+    }
+
+    pub(crate) fn format_log_record(io: &mut (dyn Write + Send), record: &Record, no_color: bool) {
```

Comment:
I have two suggestions with the `no_color` arguments: 

1. Can we utilize a positive naming convention to avoid double negatives to indicate a positive? That is, prefer something like `colored` as opposed to `no_color` to avoid NOT `no_color`?
2. In general, move away from a raw bool here. Generally, I think that [arguments convey meaning through types](https://rust-lang.github.io/api-guidelines/type-safety.html#c-custom-type). As such, we can do something like `struct Colored(bool)` at no cost and better convey the flags intent.